tasks/synopsys-detect-task/detect-task.ts

```diff
@@ -13,6 +13,7 @@ import {DetectJarConfigurationRunner} from './ts/runner/DetectJarConfigurationRu
 import {TaskResult} from 'azure-pipelines-task-lib';
 import {IDefaultScriptConfiguration} from './ts/model/IDefaultScriptConfiguration';
 import {IJarConfiguration} from './ts/model/IJarConfiguration';
+import tl = require('azure-pipelines-task-lib/task');
```

Comment:
Duplicate import of first line
`import task = require('azure-pipelines-task-lib/task');`

Reply:
removed duplicate import